sass/basics.scss

```diff
@@ -37,6 +37,33 @@ a {
   color: red;
 }
 
+.tipMessage {
+  border: 1px solid #c3e0c9;
+  background-color: #f3f9f4;
+  color: #526d8a;
+  font-size: smaller;
+  padding: 0.5em;
+  margin-bottom: 0.5em;
+}
+
+.warningMessage {
+  border: 1px solid #feedbb;
+  background-color: #fffdf7;
+  color: #526d8a;
+  font-size: smaller;
+  padding: 0.5em;
+  margin-bottom: 0.5em;
+}
+
+.dangerMessage {
+  border: 1px solid #e69a92;
+  background-color: #fff8f7;
+  color: #526d8a;
+  font-size: smaller;
```

Comment:
I think I'd keep the enclosing font size rather than using a smaller one.

Reply:
I feel that is looks a bit weird in the same size.

Here's how it looks with `small`, removed, and `medium`

<img width="1300" alt="image" src="https://user-images.githubusercontent.com/5179225/165653181-9b588de3-8120-4c0f-a777-29b9e9b8dcf3.png">